aerich/migrate.py

```diff
@@ -173,10 +173,17 @@ def _handle_indexes(cls, model: Type[Model], indexes: List[Union[Tuple[str], Ind
         ret = []
         for index in indexes:
             if isinstance(index, Index):
-                index.__hash__ = lambda self: md5(  # nosec: B303
-                    self.index_name(cls.ddl.schema_generator, model).encode()
-                    + self.__class__.__name__.encode()
-                ).hexdigest()
+                def add_hash_eq_methods_to_index(idx):
+                    def _hash(self):
+                        return hash(self.index_name(cls.ddl.schema_generator, model).encode() + self.__class__.__name__.encode())
```

Comment:
@long2ice these hash arguments are enough to ensure that every change in the index configuration will generate a different one?

Reply:
@fullonic Yes, and can be as simple as `hash(self.index_name(cls.ddl.schema_generator, model) + self.__class__.__name__)`

And if you want to add a `__eq__` function to the index instance, it is more explicit to do it like this:
```py
def eq(self, other) -> bool
       ...

if isinstance(index, Index):
    index.__hash__ = hash_func
    index.__eq__ = eq
```

Reply:
@henadzit Is it a better solution to add `__eq__`/`__hash__` into `tortoise-orm/indexes.py::Index` instead of aerich?